test/tools/OpenCover/OpenCover.psm1

```diff
@@ -353,7 +353,8 @@ function Install-OpenCover
         throw "Download failed: $packageUrl"
     }
     
-    import-module Microsoft.PowerShell.Archive
+    ## We add ErrorAction as we do not have this module on PS v4 and below. Calling import-module will throw an error otherwise.
+    import-module Microsoft.PowerShell.Archive -ErrorAction SilentlyContinue
 
```

Comment:
The commit that made this change has already been merged, can you please rebase your branch?